contracts/deploy/mainnet/138_strategist_does_buyback.js

```diff
@@ -0,0 +1,71 @@
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "138_strategist_does_buyback",
+    forceDeploy: false,
+    // forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    // proposalId: "",
+  },
+  async ({ deployWithConfirmation }) => {
+    // Proxy contracts
+    const cOUSDBuybackProxy = await ethers.getContract("BuybackProxy");
+    const cOETHBuybackProxy = await ethers.getContract("OETHBuybackProxy");
+    const cARMBuybackProxy = await ethers.getContract("ARMBuybackProxy");
+
+    // Old buyback contracts
+    const cOldOUSDBuyback = await ethers.getContractAt(
+      ["function updateBuybackSplits() external"],
```

Comment:
haven't seen this as a contract name before

Reply:
So, `getContractAt` can either take in a artifact/contract name or just ABI (in a few different formats). I had to specify the ABI explicitly here because once I deploy the contract `OETHBuyback` and `OUSDBuyback` won't have this method, which would cause an issue in the deployment file. This ensures that we can call `updateBuybackSplits` just before the upgrade (as part of the governance action)

Reply:
Ahh I see. I learned something today tnx!